app/src/main/java/com/cornellappdev/score/components/SportSelectorHeader.kt

```diff
@@ -0,0 +1,162 @@
+        Triple("Basketball", painterResource(id = R.drawable.ic_basketball), painterResource(id = R.drawable.ic_basketball_filled)),
+        Triple("Cross Country", painterResource(id = R.drawable.ic_cross_country), painterResource(id = R.drawable.ic_cross_country_filled)),
+        Triple("Football", painterResource(id = R.drawable.ic_football), painterResource(id = R.drawable.ic_football_filled))
+    )
+
+    SportSelectorHeader(sports, {}, {})
+}
+
+@Composable
+fun SportSelectorHeader(sports: List<Triple<String, Painter, Painter>>,
+                        onGenderSelected: (String) -> Unit,
+                        onSportSelected: (String) -> Unit) {
+    var selectedOption by remember { mutableStateOf("All") }
+    var selectedSport by remember { mutableStateOf(sports.first().first) }
+
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(143.dp)
+            .background(color = White)
+            .padding(start = 24.dp, bottom = 16.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.SpaceBetween
+    ) {
+        Row(
+            modifier = Modifier
+                .padding(end = 24.dp)
+                .border(
+                    border = BorderStroke(1.dp, GrayLight),
+                    shape = RoundedCornerShape(100)
+                )
+                .padding(6.dp)
+        ) {
+            GenderSelector("All", selectedOption) {
+                selectedOption = "All"
+                onGenderSelected("All")
+            }
+            GenderSelector("Men's", selectedOption) {
+                selectedOption = "Men's"
+                onGenderSelected("Men's")
+            }
+            GenderSelector("Women's", selectedOption) {
+                selectedOption = "Women's"
+                onGenderSelected("Women's")
+            }
+        }
+        LazyRow(Modifier.fillMaxWidth()) {
+            items(sports) { (sport, emptyIcon, filledIcon) ->
+                SportSelector(
+                    empty = emptyIcon,
+                    filled = filledIcon,
+                    option = sport,
+                    selectedOption = selectedSport
+                ) {
+                    selectedSport = sport
+                    onSportSelected(sport)
+                }
+
+                Spacer(modifier = Modifier.width(20.dp))
+            }
+            item { Spacer(modifier = Modifier.width(4.dp)) }
+        }
+    }
+}
+
+@Composable
+fun GenderSelector(option: String, selectedOption: String, onSelect: () -> Unit) {
+    Box(
+        modifier = Modifier
+            .clickable(onClick = {
+                onSelect()
+            })
+            .width(111.dp)
+            .background(
+                color = if (selectedOption == option) CrimsonPrimary else Color.Transparent,
+                shape = RoundedCornerShape(100) // Rounded background
+            )
+            .padding(vertical = 8.dp, horizontal = 12.dp),
```

Comment:
Hmm, I probably would've preferred the background to slide behind the gender selectors and have it animate through. I guess this is ok for now.